ddtrace/_trace/tracer.py

```diff
@@ -341,6 +347,29 @@ def deregister_on_start_span(self, func: Callable) -> Callable:
         self._hooks.deregister(self.__class__.start_span, func)
         return func
 
+    def sample_before_fork(self) -> None:
```

Comment:
Should we make this this an internal method? Would a user ever call this manually